appengine/standard/endpoints/backend/app.yaml

```diff
@@ -3,14 +3,25 @@ threadsafe: true
 api_version: 1
 
 handlers:
-# The endpoints handler must be mapped to /_ah/spi.
-# Apps send requests to /_ah/api, but the endpoints service handles mapping
-# those requests to /_ah/spi.
-- url: /_ah/spi/.*
+# The endpoints handler must be mapped to /_ah/api.
+- url: /_ah/api/.*
   script: main.api
 
 libraries:
 - name: pycrypto
   version: 2.6
-- name: endpoints
-  version: 1.0
+
+# Endpoints 1.1 uses background threads for caching and reporting to service
+# management and service control.
+manual_scaling:
```

Comment:
There are, I don't have all the details around it, but once I do I'll be sure to test/update this pull.

Maybe @tbetbetbe can offer some more information.